harvest/api/_base.py

```diff
@@ -657,7 +692,7 @@ def sell_option(self, symbol: str, quantity: int = 0, in_force: str = "gtc"):
         :returns: The result of order_option_limit(). Returns None if there is an issue with the parameters.
         """
         if quantity <= 0.0:
-            debugger.warning(
+            debugger.error(
```

Comment:
Warning is used to let the user know that `Harvest` patched the problem by itself. Logging an error is when the user needs to do something to prevent the error from happening again.